driver-core/src/main/java/com/datastax/driver/core/PoolingOptions.java

```diff
@@ -385,6 +385,10 @@ public PoolingOptions setInitializationExecutor(Executor initializationExecutor)
      * Requests the driver to re-evaluate the {@link HostDistance} (through the configured
      * {@link com.datastax.driver.core.policies.LoadBalancingPolicy#distance}) for every known
      * hosts and to drop/add connections to each hosts according to the computed distance.
+     * <p>
+     * Note that, due to backward compatibility issues, this method is not interruptible. If the
+     * caller thread gets interrupted, the method will complete and only then re-interrupt the
+     * thread (which you can check with {@code Thread.currentThread().isInterrupted()}).
```

Comment:
very small typo here: `currentThread().isInterrupted()` (comma instead of period).